### ISSUE TITLE
Visualizar Mentores

### ISSUE THREAD

**Issue by Vanessapcs** [SouJunior/products#15]
Como usuário, desejo acessar a Home Page do Portal de Mentorias, para escolher alguma funcionalidade específica.

- Contexto: 
O usuário deverá ter acesso a Home Page do Portal de Mentorias, para: (i) agendar, reagendar, cancelar a mentoria; (ii) visualizar os mentores. (ver se faltou mais alguma coisa) 

-  Critérios de Aceite / Cenários:

1. Dado que o usuário deseja acessar o Portal de Mentorias na plataforma da SouJunior;
    Quando estiver no site da SouJunior e clicar na opção de "Portal de Mentorias" ;
    Então, ele será direcionado para a Home Page.

**Comment by Vanessapcs** [comment]
@talitabitencourt o gatilho dessa US será tanto pelo "Menu" da Home do Portal, quanto pelo link de "Agendamento" encaminhado no e-mail de confirmação de cadastro. Portanto, visualizar a US de Cadastro para alterar o direcionamento do sistema após seleção do link na confirmação do e-mail.

Criar as telas.

O conteúdo acima foi colocado pelo @YuriBorsodi mas fique a vontade para editar.

**Comment by akirashimazu** [comment]
@Vanessapcs 

Posso dar continuidade nessa US? 🙂

**Comment by Vanessapcs** [comment]
@akirashimazu Sim. Só verifica se a @talitabitencourt finalizou a "página de mentores".

**Comment by talitabitencourt** [comment]
Oi @akirashimazu essa US eu já comecei a trabalhar no wireframe e como se refere ao visualizar mentores seria comigo também, acredito que a label seria a do mentor.

**Comment by akirashimazu** [comment]
@talitabitencourt @Vanessapcs 

Opa, achei que fizesse parte do módulo de mentorados, desculpe. 😅

**Comment by alanagarbuli** [comment]
@DanielDefault1 Dani, pode validar essa US para falarmos sobre no 1:1 de hoje?

**Comment by DanielDefault1** [comment]
@alanagarbuli Lana, revisei e vou sugerir algumas alterações nela durante o nosso 1:1. Depois eu coloco elas aqui pra que você possa consultar no futuro!

**Comment by DanielDefault1** [comment]
Estrutura pra escrever user statement: Como [persona/usuário], preciso de [ação que o usuário realiza com valor perceptível], para [resultado a ser atingido com a ação]
Exemplo: Como mentorado, desejo encontrar um Mentor com as especialidades e habilidades que quero aprimorar/desenvolver, para obter sessões de mentoria com aconselhamento

Deixar claro na descrição da US que o resultado esperado do desenvolvimento é a imagem 2, a imagem 1 e 3 estão ali apenas para orientação do fluxo da jornada do usuário.

Escrever os critérios de aceite baseados na imagem abaixo, usando um esquema similar ao que temos como BDD
![Image](https://github.com/SouJunior/products/assets/142111521/c5cf56be-4a3f-4afa-8f04-412bbe73a621)

**Comment by alanagarbuli** [comment]
@DanielDefault1 US finalizada! Poderia validar, por favor?!

**Comment by DanielDefault1** [comment]
@alanagarbuli ficou ótima, muito boa a organização!

Só coloque uma última observação no cenário de filtros: mencione que ambos os filtros devem se comportar como um checkbox.
Checkbox é um elemento de programação front-end que permite ao usuário selecionar mais de um filtro ao mesmo tempo ele é o oposto do que chamamos de radioButton (que só permite selecionar um filtro por vez).
Ou seja, em resumo, queremos que o usuário possa selecionar mais de uma opção de filtro durante a busca.

**Comment by alanagarbuli** [comment]
@DanielDefault1 eu segui conforme aquele resuminho no figma do Cris que falamos ontem.
Alinharei a mudança com ele hoje e já corrigi o critério de aceite!

**Comment by alanagarbuli** [comment]
@luizpbello A ideia é que a página fique vazia com a informação de que não localizamos mentores de acordo com o filtro selecionado. 

[Cenário: mentores não localizados]
Dado que o mentorado queira pesquisar por mentores e não os localiza
Quando fizer uso dos filtros
Então apresentar a tela vazia com a seguinte mensagem: Ops! Não encontramos mentores de acordo com os filtros selecionados.

@cristomazini consegue criar a tela pro @luizpbello tocar?!

@luizpbello esse não possui links! É apenas o topo de página mesmo. 
E sim! O ideal é que apareçam apenas mentores que estão com o cadastro completo pois assim sabemos que é uma pessoa que está pronta para seguir com as mentorias. Um cadastro incompleto entendo que é um mentor indisponível pois não conseguirei saber quais as skills domina e também não terá o calendly configurado. Faz sentido?!

**Comment by alanagarbuli** [comment]
@luizpbello tudo bem?!

A opção **VER TODOS** da Home Page do cenário 2 não está clicável.
É importante pois é ali que o usuário será direcionado para a página de mentores construída nessa US.

**Comment by alanagarbuli** [comment]
@raiane-oliveira tudo bem?!

A opção VER TODOS da Home Page do cenário 2 não está clicável.
É importante pois é ali que o usuário será direcionado para a página de mentores construída nessa US.
Consegue corrigir, por favor?!

**Comment by raiane-oliveira** [comment]
Oii @alanagarbuli! Acabei de consertar isso e já fiz uma PR

**Comment by alanagarbuli** [comment]
@raiane-oliveira tudo bem?!
Consegue incluir o rodapé na página? Para que possamos acessar as redes sociais, áreas logadas e FAQ.
Obrigada!

**Comment by raiane-oliveira** [comment]
@alanagarbuli acabei de ajeitar. Tu consegue visualizar por esse link enquanto a PR ainda não foi aprovada -> https://mentores-frontend-five.vercel.app/mentores

**Comment by andremomesso** [comment]
Estou encerrando esta demanda, pois a mesma já está em Produção.
